api/debug.go

```diff
@@ -9,3 +9,14 @@ type Debug struct {
 type DebugPatch struct {
 	IsDebug bool `jsonapi:"attr,isDebug"`
 }
+
+// DebugLog is the API message for debug log.
+type DebugLog struct {
+	ID          int    `jsonapi:"primary,log"`
+	RecordTs    int64  `jsonapi:"attr,recordTs"`
+	Method      string `jsonapi:"attr,method"`
+	RequestPath string `jsonapi:"attr,requestPath"`
+	Role        Role   `jsonapi:"attr,role"`
+	Error       string `jsonapi:"attr,error"`
+	StackTrace  string `jsonapi:"attr,stackTrace"`
```

Comment:
These lines are identical to `ErrorRecord` due to https://github.com/google/jsonapi/issues/74. 
Otherwise, we can have
```go
type DebugLog struct {
    ID int `jsonapi:"primary,log"`
    Record ErrorRecord `jsonapi:"attr,errorRecord"`
}
```
which looks much better to me.

Reply:
The google/jsonapi issue above seems to be mutation API only. As you suggested, let's combine the structures if possible.

Reply:
SG.